### PR TITLE
Creating Your First Android Application on Android Studio.

### LINKED ISSUE #953: Creating Your First Android Application on Android Studio.
### Brief Summary: 
Mobile Apps are still being used all over the 'world. This article will guide the reader on how to create their first android application until they run it on their phones. It will contain a step-by-step process, code and images on how to come up with the application.

### Key Takeaways: 
1. Android Studio Setup.

2. Designing the Layout.

3. Writing the code in Java.

4. Running and testing the app on your phone.

### References: 
N/A.

### PR DESCRIPTION
**Pull request by bluemutheu**
### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #953

### CODE REVIEW

**Comment by kayere** [comment]
@bluemutheu - This PR contains two articles. Could you please remove the irrelevant one so that I can begin the review.

**Comment by bluemutheu** [comment]
@kayere It has been done

File: articles/first-android-app/index.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+> NOTE: Download according to the version of windows you are running, 64bit or 32bit.
+ According to Google, you need 2GB of RAM and above (4GB is recommended) to run Android Studio and Java since they are memory consuming and will slow down your PC.
+
+### New terminologies
+ - **Method** - A block of code that only runs when called and performs an action.
+
+ - **Activity** - This is one screen on an Android App's user interface.
+
+ - **Fragment** - It is a part of an Activity.
```
Comment (kayere) [inline articles/first-android-app/index.md:51]: This definition is vague. Consider using something like this *'A Fragment represents a reusable portion of your app's UI'*

File: articles/first-android-app/index.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+```
+
+### Step 3. Write the Java Code
+We find the Java classes for the Activities in the ***Java Folder*** -> then the first folder with the ***Application's package name***
+
+Java code for the First Activity:
+- In the first activity outside the `onCreate` method, call the `onClick` you created in the XML on the button.
+- Inside this method, convert the text input from the user to a String.
+- Then create an extra that will facilitate parsing of the data.
+- Finally, create an intent for Activity navigation.
```
Comment (kayere) [inline articles/first-android-app/index.md:163]: Please explain what an intent is. It is obviously a new term for a beginner.

**Review by kayere — COMMENTED** [review]
Hello @bluemutheu - Great article, quite informative. Please make the following changes and the article will be good to go.

**Comment by kayere** [comment]
And I have noticed that your branch has conflicts with the base branch. 
Please resolve the conflict.

**Comment by bluemutheu** [comment]
> And I have noticed that your branch has conflicts with the base branch.
> Please resolve it.

@kayere It's not giving me the option to resolve the conflicts due to write access

**Comment by kayere** [comment]
I too can not resolve the conflict. I'd suggest you create a new PR with only the files needed for the article. That should work.
Notify me when you are done so that I can go through it one last time.

**Comment by bluemutheu** [comment]
@kayere Okay, noted

**Comment by bluemutheu** [comment]
@kayere Should I close this PR?

**Comment by kayere** [comment]
You can.

**Pull request closed without merging**